src/lib/utils.ts

```diff
@@ -80,6 +78,65 @@ export const metricFormatter = (
   return value.toFixed(2);
 };
 
+/**
+ * delay a given number of milliseconds
+ * @param ms:number - number of milliseconds to delay
+ * @returns Promise<void>
+ */
+export async function delay(ms: number): Promise<void> {
```

Comment:
What is this used for? I don't see any usages and if I did I would likely complain